open-problems.md

```diff
@@ -56,4 +56,10 @@ and [AutoNAT](https://docs.libp2p.io/concepts/nat/#hole-punching-stun#autonat).
 The main focus of this topic is on researching novel ICE-complementary NAT traversal techniques that are decentralized and have desirable privacy and anonymity properties.
 The topic also comprises integrating and specifying ICE for [Waku v2](https://rfc.vac.dev/spec/10/).
 
+# Further Research Collaboration Opportunities
```

Comment:
My comment is about the section above where the ongoing research on discv5 is mentioned.
```
A new version of discv5 that is both efficient and provides eclipse mitigation is [currently being researched](https://github.com/harnen/service-discovery-paper).
```
The link directs to a GH repo which is not very descriptive, is there any whitepaper that we can link instead?